web/api/v1/api.go

```diff
@@ -78,6 +85,15 @@ var corsHeaders = map[string]string{
 	"Access-Control-Expose-Headers": "Date",
 }
 
+var remoteReadQueries = prometheus.NewGaugeVec(
+	prometheus.GaugeOpts{
+		Namespace: namespace,
+		Subsystem: subsystem,
+		Name:      "remote_read_queries",
+		Help:      "The current number of remote read queries being executed or waiting.",
+	},
+	[]string{urlLabel})
```

Comment:
No need to make a new variable. Just out `[]string{"url"}` here.

Reply:
alright